www/video/compare.php

```diff
@@ -722,6 +722,11 @@ function DisplayGraphs() {
     $progress_end = 0;
     foreach($tests as &$test) {
         $hasStepResult = array_key_exists('stepResult', $test) && is_a($test['stepResult'], "TestStepResult");
+        if ($hasStepResult &&

```

Comment:
Is it worth adding all of the hero times?  Not sure how often they differ but it could be good to expose here since there is more room for metrics in this UI than the results view.

Reply:
Yes, it might make more sense to list the individual times on this chart rather than the aggregates. Will update.

Reply:
While this feature is still behind a flag, I'm inclined to rename the three default heroes:

* h1 -> Heading
* biggest_img -> Image
* bg_img -> BackgroundImage

That makes the names a bit friendlier in the UI, because we can just prepend "Hero" to them. "Hero Heading" reads better than "Hero h1" IMO. Thoughts?